tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot

```diff
@@ -132,33 +131,29 @@ Connectivity Bridge to Public
     Log To Console  Installer completed successfully: %{VCH-NAME}
 
     Log To Console  Creating public container.
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d --net=vm-network --name p1 busybox /bin/top
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d --net=vm-network -p 8000 --name p1 busybox nc -l -p 8000
     Should Be Equal As Integers  ${rc}  0
 
-    Log To Console  Starting public container
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} start p1
+    Log To Console  Getting IP for public container
+    ${ip}=  Run  docker %{VCH-PARAMS} inspect --format '{{range .NetworkSettings.Networks}}{{.IPAddress }}{{end}}' p1
+
+    Log To Console  Connecting to container on external network from container bridged network
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run --net bridge busybox nc ${ip} 8000
```

Comment:
if you are going to switch from ping to netcat, should we be piping stuff into it and verify it shoes up on the other end?